mantidproject/mantid
# New CHESS IDF file and Parameter file

**Description of work.**

New CHESS IDF file
Changed MaskBTP algorithm
Changed Q/E coverage and DGSPlanner interfaces to work with CHESS

**To test:**

Make sure that `LoadEmptyInstrument` with CHESS works fine.
Make sure that MaskBTP, Q/e coverage and DGSPlanner work fine.



*There is no associated issue.*




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
